README.md

```diff
@@ -36,24 +36,18 @@ In the demo client [repository](https://github.com/tecton-ai/TectonClientDemo) u
 jar that you generate from this repo using `./gradlew clean build`.
 
 1. Change the dependencies target to this and point the files attribute to your java client jar.
-2. Add all dependencies from `tecton-http-client-java` except `testImplementation` to `build.gradle` in the demo client repository.
+2. Add the okhttp3 dependency from `tecton-http-client-java` to `build.gradle` in the demo client repository. This is
+   necessary because it is an API dependency, but JAR files don't include information about transitive dependencies.
+   Published libraries on Maven don't have this issue.
```

Comment:
are the dependencies below (line 44+) not needed anymore ?

Reply:
I tested it without them (with a local jar) and it worked!

Reply:
But I'll leave them with a comment.